app/src/main/java/com/example/next_show/fragments/FeedFragment.java

```diff
@@ -48,21 +55,29 @@
     public static final int NOT_FOUND = -1;
 
     // genres for shows
-    public static final String ACTION = "action";
-    public static final String COMEDY = "comedy";
-    public static final String DRAMA = "drama";
-    public static final String ALL_SHOWS = "all";
+    private static final String ACTION = "action";
+    private static final String COMEDY = "comedy";
+    private static final String DRAMA = "drama";
+    private static final String ALL_SHOWS = "all";
+
+    // filters
+    public static final String GENRE = "genre";
+    public static final String NETWORK = "network";
+    public static final String YEAR = "year";
```

Comment:
I would only define these in one place.  You can make them public in the ShowFilter class